### PR TITLE
two_unets

### PR DESCRIPTION
**Pull request by kdziedzic66**

### CODE REVIEW

File: pipeline_config.py [hunk 1]
```diff
@@ -98,9 +98,103 @@
...
+            'experiment_timing': {'batch_every': 0,
+                                  'epoch_every': 1},
+            'validation_monitor': {'epoch_every': 1},
+            'neptune_monitor': {    'model_name': 'unet_mask',
+                                    'image_nr': 4,
+                                'image_resize': 0.2},
+            'early_stopping': {'patience': params.patience},
+        },
+    },
+    'unet_contour': {
```
Comment (jakubczakon) [inline pipeline_config.py]: all this should be defined in the neptune.yaml with a seperate prefix. For example 'unet_contour_n_filters'

File: pipelines.py [hunk 2]
```diff
@@ -92,6 +92,65 @@ def unet_multitask(config, train_mode):
                   cache_dirpath=config.env.cache_dirpath)
     return output
 
+def two_unets(config, train_mode):
```
Comment (jakubczakon) [inline pipelines.py]: this name is not very descriptive. Maybe something like `two_unet_specialists` or `unet_mask_contour_specialists` would be better.

File: neptune.yaml [hunk 3]
```diff
@@ -52,7 +52,7 @@ parameters:
   image_channels: 3
 
 # U-Net parameters
```
Comment (jakubczakon) [inline neptune.yaml]: @kdziedzic66 both unets should be defined here. `pipelines_config.py` is to a large extend just a formatter of this config.

File: pipelines.py (hunk 2 above)
Comment (kamil-kaczmarek) [reply pipelines.py]: I agree with @kdziedzic66, that good name for it is: `two_unet_specialists`
@jakubczakon

**Pull request merged**